### PR TITLE
[REVIEW] Update Conda Environment for Pythoin 3.8 and CUDA 11

### LINKED ISSUE #1023: [BUG] Conda CIDA 11.0 env file
Now that we support CUDA 11.0, we need a conda environment YML file.

### PR DESCRIPTION
**Pull request by BradReesWork**
closes #1023

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: conda/environments/cugraph_dev_cuda10.1.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ dependencies:
 - clang=8.0.1
 - clang-tools=8.0.1
 - cmake>=3.12
-- python>=3.6,<3.8
+- python>=3.6,<3.9
```
Comment (seunghwak) [inline conda/environments/cugraph_dev_cuda10.1.yml:23]: So, are we supporting one more python version (adding 3.8 without dropping 3.6)?

Comment (seunghwak) [reply conda/environments/cugraph_dev_cuda10.1.yml:23]: If yes, this looks good to me.

Comment (BradReesWork) [reply conda/environments/cugraph_dev_cuda10.1.yml:23]: we are adding 3.8.  I have left 3.6 in there for now, but it will be dropped soon

**Pull request merged**